show/main.py

```diff
@@ -1181,16 +1181,58 @@ def vlan():
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def brief(verbose):
     """Show all bridge information"""
-    cmd = "sudo brctl show"
-    run_command(cmd, display_cmd=verbose)
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    header = ['VLAN ID','IP Address','Ports','DHCP Helper Address']
+    body = []
+    vlan_keys = []
 
-@vlan.command()
-@click.argument('bridge_name', required=True)
-@click.option('--verbose', is_flag=True, help="Enable verbose output")
-def id(bridge_name, verbose):
-    """Show list of learned MAC addresses for particular bridge"""
-    cmd = "sudo brctl showmacs {}".format(bridge_name)
-    run_command(cmd, display_cmd=verbose)
+    # Fetching data from config_db for VLAN, VLAN_INTERFACE and VLAN_MEMBER
+    vlan_dhcp_helper_data = config_db.get_table('VLAN')
+    vlan_ip_data = config_db.get_table('VLAN_INTERFACE')
+    vlan_ports_data = config_db.get_table('VLAN_MEMBER')
+
+    vlan_keys = natsorted(vlan_dhcp_helper_data.keys())
+
+    # Defining dictionaries for DHCP Helper address, Interface Gateway IP and VLAN ports
+    vlan_dhcp_helper_dict = {}
+    vlan_ip_dict = {}
+    vlan_ports_dict = {}
+
+    # Parsing DHCP Helpers info
+    for key in natsorted(vlan_dhcp_helper_data.keys()):
+        try:
+            if vlan_dhcp_helper_data[key]['dhcp_servers']:
+                vlan_dhcp_helper_dict[str(key.strip('Vlan'))] = vlan_dhcp_helper_data[key]['dhcp_servers']
+        except KeyError:
+            vlan_dhcp_helper_dict[str(key.strip('Vlan'))] = " "
+            pass
+
+    # Parsing VLAN Gateway info
+    for key in natsorted(vlan_ip_data.keys()):
+        interface_key = str(key[0].strip("Vlan"))
+        interface_value = str(key[1])
+        if interface_key in vlan_ip_dict:
+            vlan_ip_dict[interface_key].append(interface_value)
+        else:
+            vlan_ip_dict[interface_key] = [interface_value]
+
+    # Parsing VLAN Ports info
+    for key in natsorted(vlan_ports_data.keys()):
+        ports_key = str(key[0].strip("Vlan"))
+        ports_value = str(key[1])
+        if ports_key in vlan_ports_dict:
+            vlan_ports_dict[ports_key].append(ports_value)
+        else:
+            vlan_ports_dict[ports_key] = [ports_value]
+
+    # Printing the following dictionaries in tablular forms: vlan_dhcp_helper_dict={}, vlan_ip_dict = {}, vlan_ports_dict = {}
+    for key in natsorted(vlan_dhcp_helper_dict.keys()):
+        dhcp_helpers = ','.replace(',','\n').join(vlan_dhcp_helper_dict[key])
+        ip_address = ','.replace(',','\n').join(vlan_ip_dict[key])
+        vlan_ports = ','.replace(',','\n').join((vlan_ports_dict[key]))
+        body.append([key,ip_address,vlan_ports,dhcp_helpers])
+    click.echo(tabulate(body, header, tablefmt = "grid"))
```

Comment:
We follow pep8 style guidelines in order to ensure consistent style throughout our Python files. For this case, a keyword argument assignment, pep8 guidelines state that there should be no spaces around the `=`, whereas in regular variable assignments, there should be spaces. I suggest you pass your code through this website: http://pep8online.com to check your code. There are also a few issues with missing spaces after commas. With regards to line length, you don't need to follow the 80-character rule, because we all use wider monitors these days. We try to keep lines under 100 chars. Just use your best judgement when to break lines for best readability.

Reply:
Thanks Joe. I have updated the code as much as I can as per PEP8 reco

Reply:
Thanks, Deva!